Automattic/wp-calypso
# Signup: Update site-or-domain step

This pull request fixes #12635 by updating the `site-or-domain` step with new domains, and updating the tooltip for custom domains on the plans page to show the selected domain.

<img width="579" alt="screen shot 2017-04-26 at 11 25 35" src="https://cloud.githubusercontent.com/assets/275961/25430114/4efac4c0-2a73-11e7-8a7d-5ab9f98b3a05.png">
<img width="818" alt="screen shot 2017-04-26 at 11 25 24" src="https://cloud.githubusercontent.com/assets/275961/25430115/4efe9046-2a73-11e7-83eb-49091b37d296.png">

This also updates the mobile view:
<img width="464" alt="screen shot 2017-04-28 at 14 27 52" src="https://cloud.githubusercontent.com/assets/275961/25530658/f6685982-2c1e-11e7-9045-1346cc77773c.png">

#### Testing instructions
  
1. Run `git checkoutupdate/site-or-domain-step` and start your server, or open a [live branch](https://delphin.live/?branch=update/site-or-domain-step)
2. Open the [`Domain first signup` page](http://calypso.localhost:3000/start/domain-first/site-or-domain?new=adfgadfgd23453.com)
3. Check that you see the new design for this screen
2. Open the [`Regular signup` page](http://calypso.localhost:3000/start)
2. Select a domain name
3. Check that you see the domain name in the tooltip on the plans page
  
#### Reviews
  

@Automattic/sdev-feed
